### ISSUE TITLE
App crash with no explication during Quick configuration

### ISSUE THREAD

**Issue by Olafr9500** [cypress-io/cypress#22307]
### Current behavior

1st setup:
After clicking "continue" on the "Configuration files" page, the application closes and the following message is displayed:

```bash
Opening Cypress...

UNKNOWN: unknown error, watch
Error: UNKNOWN: unknown error, watch
    at FSEvent.FSWatcher._handle.onchange (node:internal/fs/watchers:204:21)
```

If I reopen the application, I return to the "Welcome to Cypress!" page and I have a lot of alerts that say:
Warning: Unexpected Internal Error

We encountered an unexpected internal error. Please check GitHub or open a new issue if you don't see one already with the details below:

clicking on "E2E Testing" closes the application and displays the following messages:

```bash
UNKNOWN: unknown error, watch
Error: UNKNOWN: unknown error, watch
    at FSEvent.FSWatcher._handle.onchange (node:internal/fs/watchers:204:21)
```

### Cypress Version

10.1.0

### Package Manager

npm

### Operating system

Windows

### Other

Dependencies :  {
    "@babel/preset-env": "^7.18.2",
    "@fullhuman/postcss-purgecss": "^4.1.3",
    "@olafr/modulejs": "^1.2.3",
    "@rollup/plugin-commonjs": "^22.0.0",
    "@rollup/plugin-json": "^4.1.0",
    "@rollup/plugin-node-resolve": "^13.3.0",
    "autoprefixer": "^10.4.7",
    "bootstrap": "^5.1.3",
    "bootstrap-icons": "^1.8.1",
    "clean-css-cli": "^5.6.0",
    "cypress": "^10.1.0",
    "eslint": "^8.10.0",
    "eslint-plugin-cypress": "^2.12.1",
    "fantasticon": "^1.2.3",
    "jsbarcode": "^3.11.5",
    "minify": "^8.0.4",
    "npm-run-all": "^4.1.5",
    "postcss": "^8.4.14",
    "purgecss": "^4.1.3",
    "qrcode": "^1.5.0",
    "rollup": "^2.75.6",
    "rollup-plugin-babel": "^4.4.0",
    "rollup-plugin-terser": "^7.0.2",
    "sass": "^1.49.9",
    "terser": "^5.14.0",
    "watch": "^1.0.2"
  }

cypress is also install with "npm -g"

**Comment by Olafr9500** [comment]
Good news, I removed cypress globally, and I have no more problems.
It is possible that my old global configuration has disturbed the configuration (it was in version 6.9.0).
I will try to reproduce the problem, because I deleted the global installation before seeing your return.

**Comment by jennifer-shehane** [comment]
Yeah, we definitely recommend installing Cypress per project via a package manager.

**Comment by Olafr9500** [comment]
Sorry, I can't reproduce the problem.

I close the topic.

**Comment by NabeelFz** [comment]
We encountered an unexpected internal error. Please check GitHub or open a new issue if you don't see one already with the details below:

clicking on "E2E Testing" closes the application and displays the following messages:

UNKNOWN: unknown error, watch
Error: UNKNOWN: unknown error, watch
    at FSEvent.FSWatcher._handle.onchange (node:internal/fs/watchers:204:21)

**Comment by rammi22** [comment]
I face the same error...
> We encountered an unexpected internal error. Please check GitHub or open a new issue if you don't see one already with the details below:
> 
> clicking on "E2E Testing" closes the application and displays the following messages:
> 
> UNKNOWN: unknown error, watch Error: UNKNOWN: unknown error, watch at FSEvent.FSWatcher._handle.onchange (node:internal/fs/watchers:204:21)


> Yeah, we definitely recommend installing Cypress per project via a package manager.

At least in my project, already done:
```
{
  "name": "npm_test",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "cypress": "cypress open"
  },
  "author": "",
  "license": "ISC",
  "devDependencies": {
    "cypress": "^12.17.4"
  }
}
```

Any advances or suggestions?

**Comment by MikeMcC399** [comment]
@rammi22

> Any advances or suggestions?

This issue is already closed and it is not the same configuration as you have.

- If you think you have found a bug, you should open a [new installation issue](https://github.com/cypress-io/cypress/issues/new?assignees=&labels=topic%3A+installation&projects=&template=3-install-issue.yml) and provide all the details requested. 
- If you would like to ask for help from the experienced Cypress community, then you can reach out to them via the Discord chat [![Discord chat](https://img.shields.io/badge/chat-on%20Discord-brightgreen)](https://on.cypress.io/discord)
(click on button)

Make sure you do not have cypress installed globally. So for instance
`npm uninstall cypress -g`
